x-pack/plugins/security_solution/public/common/components/event_details/alert_summary_view.tsx

```diff
@@ -120,23 +144,35 @@ const getSummaryRows = ({
 
   return data != null
     ? tableFields.reduce<SummaryRow[]>((acc, item) => {
+        const initialDescription = {
+          contextId: timelineId,
```

Comment:
Depending on what the draggable removal situation looks like we may end up having to remove a lot of this logic in a follow up PR